src/operator.c

```diff
@@ -178,7 +179,25 @@ static PetscErrorCode AddCeedOperators(Operator *op) {
   PetscFunctionReturn(PETSC_SUCCESS);
 }
 
-static PetscErrorCode AddPetscOperators(Operator *op) {
+static PetscErrorCode AddCeedSedimentOperators(Operator *op) {
+  PetscFunctionBegin;
+  PetscCheck(PETSC_FALSE, PETSC_COMM_WORLD, PETSC_ERR_USER, "Extend code to add CEED version of sediment operators");
+  PetscFunctionReturn(PETSC_SUCCESS);
+}
+
+static PetscErrorCode AddCeedOperators(Operator *op) {
+  PetscFunctionBegin;
+
+  if (!op->config->physics.sediment.num_classes) {
+    PetscCall(AddCeedFlowOperators(op));
+  } else {
+    PetscCall(AddCeedSedimentOperators(op));
```

Comment:
So the sediment operators also incorporate the flow equations? Is this because the Riemann solver we're using applies to the entire system, as opposed to solving for h, hu, hv in a "flow" Riemann solver and using an upwinding/advection method for the sediment classes?

Reply:
Yes, the Riemann solver in SD is solving for all of the prognostic variables (i.e., h/hu/hv/hci).